crypto/armcap.c

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2011-2022 The OpenSSL Project Authors. All Rights Reserved.
+ * Copyright 2011-2023 The OpenSSL Project Authors. All Rights Reserved.
```

Comment:
No need to update copyrights, we've a script that does this as part of the release.
Leave it in, it's harmless.

Reply:
(old habits die hard!)